src/test/java/com/commercetools/sync/sdk2/categories/utils/CategoryTransformUtilsTest.java

```diff
@@ -0,0 +1,70 @@
+package com.commercetools.sync.sdk2.categories.utils;
+
+import static com.commercetools.sync.sdk2.products.ProductSyncMockUtils.readObjectFromResource;
```

Comment:
Maybe it would make sense to create common mock utils with this method.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/973/commits/a8efb8c4ebe34c70ac381344387eb226045c70ff